src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandParserSupport.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    public class CommandParserSupport
+    {
+        private static IEngineEnvironmentSettings _environment;
+        private static SettingsLoader _settingsLoader;
+
+        public static void Setup(IEngineEnvironmentSettings environment)
+        {
+            _environment = environment;
+            _settingsLoader = (SettingsLoader)_environment.SettingsLoader;
+        }
+
+        public static Command CreateNewCommandWithoutTemplateInfo(string commandName) => GetNewCommand(commandName, NewCommandVisibleArgs, NewCommandHiddenArgs, DebuggingCommandArgs);
+
+        private static Command GetNewCommand(string commandName, params Option[][] args)
+        {
+            Option[] combinedArgs = CombineArrays(args);
+
+            return Create.Command(commandName,
+                           "Initialize .NET projects.",
```

Comment:
👍 Using the existing LocalizableStrings.CommandDescription which is "Template Instantiation Commands for .NET Core CLI"